src/help/components/contextual/styles/contextual-help.module.scss

```diff
@@ -0,0 +1,26 @@
+@import 'franklin-sites/src/styles/colours';
+
+$margin-offset: 1rem;
```

Comment:
You could use the margin value from franklin, it happens to be the same

Reply:
Ah, but I see it's to cancel the normal margin, OK, I don't know then

Reply:
Actually hadn't realised the sliding panel had a `title` prop which does what we want (apart from colour)...